src/workshops/node-postgres/index.md

```diff
@@ -76,27 +76,47 @@ This script executes the SQL commands inside `/database/init.sql` against your D
+```
+postgres://learn_pg_user:@localhost:5432/learn_pg
+```
 
-Take a look at `/database/connection.js`. We import the `pg` library, then connect to the right DB by passing in a "connection string" (a URL identifying the DB and user).
+You _could_ hard-code this URL into our app code, but this address is only correct for the database running on your computer. When someone else clones your repo they'll have their own DB set up.
 
-{% disclosure "More info about the connection string" %}
+### Environment variables
 
-The connection string is a URL pointing to your exact Postgres DB, including the user and password. It is structured like this:
+It's best to read configuration like this from an "environment variable" (env vars). This is a value set in the shell before you start your application. For example it's common to read the `PORT` env var to know what port your app should listen on.
 
+Take a look at the `/database/connection.js` file. It imports the `pg` library, then connects to the right DB by passing in the connection string. It reads this from the `DATABASE_URL` env var, which means we must make sure this is set before starting our server.
+
+Rather than type `DATABASE_URL=postgres://... npm run dev` every time, we can rely on the popular `dotenv` library. This allows us to define env vars in a file named `.env`. We gitignore this file so each person who clones the repo can make their own with their own personal DB URL.
+
+First install `dotenv` as a dev dependency:
+
+```shell
+npm install --save-dev dotenv
 ```
-postgres://username:password@localhost:5432/database_name
-```
 
-We read it from an environment variable in `connection.js`—this is because we want the DB password to stay secret, and so that different environments can connect to different DBs. E.g. we want to use a local DB during development but one on Heroku when we deploy our app.
+Then create a `.env` file at the root of your project containing:
+
+```shell
+DATABASE_URL='postgres://learn_pg_user:@localhost:5432/learn_pg'
+```
 
-When we run `npm run dev` to start the server it will execute `nodemon -r dotenv/config server.js`. This tells Node to use the [`dotenv`](https://www.npmjs.com/package/dotenv) library to load env vars _before_ our server starts. This will read the `.env` file and load all the env vars automatically.
+Then change your `dev` npm script in the `package.json` file to this:
 
-{% enddisclosure %}
+```
+"nodemon -r dotenv/config server.js"
+```
 
-The `/database/connection.js` file exports a "pool" of connections for our app to use. This helps handle multiple requests at the same time. We can import the `db` variable in any other file to send requests to the DB.
+The `-r dotenv/config` bit tells the `dotenv` library to read the `.env` file and pass all the values inside it to your application. They will then be accessible via `process.env.VAR_NAME`.
```

Comment:
"you can access them in your code via `process.env.VAR_NAME`"

just to be more explicit since there is quite a bit of jumping between terminal, package.json, and application code here.

Reply:
Looks good otherwise! Definitely think "don't install for them" is a good idea😅